### PR TITLE
BER DIONI almost done

### PR DESCRIPTION
**Pull request by dionifss**
I will keep working a bit more tommorow and repush it ;)

### CODE REVIEW

**Comment by dionifss** [comment]
All done less the last one ;), if the push is not correct just let me know

File: src/functions-and-arrays.js [hunk 1]
```diff
@@ -1,73 +1,500 @@
-// Iteration #1: Find the maximum
+function maxOfTwoNumbers(n1, n2) {
+  if (n1 > n2) {
+    return n1;
+  } else if (n2 > n1) {
+    return n2;
+  } else {
+    return n1;
```
Comment (fonsogms) [inline src/functions-and-arrays.js]: You could have saved and else if like this: 
if (n1 > n2) {
    return n1;
  } else{return n2}

File: src/functions-and-arrays.js [hunk 2]
```diff
@@ -1,73 +1,500 @@
...
+
+const mixedArr = [6, 12, "miami", 1, true, "barca", "200", "lisboa", 8, 10];
+
+function sum(mixedArr) {
+  let sumatory2 = 0;
+  for (i = 0; i < mixedArr.length; i++) {
+    if (typeof mixedArr[i] === "object") {
+      return "Unsupported data type sir or ma'am";
+    }
+    if (typeof mixedArr[i] === "boolean") {
```
Comment (fonsogms) [inline src/functions-and-arrays.js]: no need to at this statement, as true + a number is equal to= 1+number :)

File: src/functions-and-arrays.js [hunk 3]
```diff
@@ -1,73 +1,500 @@
...
-const wordsArr = ['seat', 'correspond', 'linen', 'motif', 'hole', 'smell', 'smart', 'chaos', 'fuel', 'palace'];
+function averageWordLength(wordsArr) {
+  let sumatory4 = 0;
+  for (i = 0; i < wordsArr.length; i++) {
+    sumatory4 += wordsArr[i].length;
+  }
+  if (sumatory4 == 0) {
+    return null;
+  }
+  sumy1 = sumatory4 / wordsArr.length;
```
Comment (fonsogms) [inline src/functions-and-arrays.js]: avoid creating a global variable that is being used by more than one function, and create that variable inside of the function instead

**Pull request closed without merging**